stockflux-securities-master/src/reducers/inputFormReducer.js

```diff
@@ -0,0 +1,24 @@
+import { InputFormState } from "../enums";
+
+export function inputFormReducer(state, action) {
+  switch (action.type) {
+    case InputFormState.LOADING:
+      return { ...state, fetchStatus: InputFormState.LOADING, messages: [] };
+    case InputFormState.LOADING:
+      return { ...state, fetchStatus: InputFormState.SENDING, messages: [] };
+    case InputFormState.ERROR:
+      return {
+        ...state,
+        fetchStatus: null,
```

Comment:
See comment below regarding setting `fetchStatus` to null. Note, that it is set to null in the case below as well.

Reply:
I have addressed this in both reducers by setting it to an enum `TableState.COMPLETED` and `InputFormState.COMPLETED`